modules/ppcp-button/resources/js/modules/ContextBootstrap/SingleProductBootstap.js

```diff
@@ -217,6 +217,11 @@ class SingleProductBootstap {
     }
 
     simulateCart() {
+        // Check of cart simulation is enabled.
```

Comment:
mistype, "if"

not sure though if this comment adds something useful, it is almost the same as the code line below.